frontend/resourceadm/pages/ResourcePage/ResourcePage.tsx

```diff
@@ -239,15 +197,45 @@ export const ResourcePage = (): React.JSX.Element => {
   return (
     <div className={classes.resourceWrapper}>
       <div className={classes.leftNavWrapper}>
-        <LeftNavigationBar
-          upperTab='backButton'
-          tabs={getTabs()}
-          backLink={getResourceDashboardURL(org, app)}
-          backLinkText={t('resourceadm.left_nav_bar_back')}
-          selectedTab={
+        <StudioContentMenu.Static
+          onChangeTab={(tabId: NavigationBarPage) => {
+            if (tabId !== 'back') {
+              navigateToPage(tabId);
+            }
+          }}
+          selectedTabId={
             currentPage === migrationPageId && !isMigrateEnabled() ? aboutPageId : currentPage
           }
-        />
+        >
+          <StudioContentMenu.LinkTab
+            tabId={'back'}
+            tabName={t('resourceadm.left_nav_bar_back')}
+            icon={<ArrowLeftIcon />}
+            renderTab={(props) => <Link to={getResourceDashboardURL(org, app)} {...props} />}
+          />
+          <StudioContentMenu.ButtonTab
+            tabId={'about'}
+            tabName={t('resourceadm.left_nav_bar_about')}
+            icon={<InformationSquareIcon />}
+          />
```

Comment:
Maybe we can create a list of objects and then just create one map function to display the buttons? Something like this maybe 😄 

```tsx
const contentMenuItems: StudioContentMenuButtonTabProps[] = [{
   tabId: 'about', 
   tabName: {t('resourceadm.left_nav_bar_about')}
   icon: {<InformationSquareIcon />}
}, {
   tabId='policy'
    tabName={t('resourceadm.left_nav_bar_policy')}
    icon={<GavelSoundBlockIcon />}
}, {
   ...
}]

...
   contentMenuItems.map((menuItem: StudioContentMenuButtonTabProps) => (
      <StudioContentMenu.ButtonTab {...menuItem} />
   )
...
```